libsquoosh/src/codecs.ts

```diff
@@ -1,5 +1,13 @@
 import { promises as fsp } from 'fs';
 import { instantiateEmscriptenWasm, pathify } from './emscripten-utils.js';
+import { threads } from 'wasm-feature-detect';
+import { cpus } from 'os';
+
+// We use `navigator.hardwareConcurrency` for Emscripten’s pthread pool size.
+// This is the only workaround I can get working without crying.
```

Comment:
To maintain consistency with the previous sentence
```suggestion
// This is the only workaround we can get working without crying.
```